common-docs/python/functions.md

```diff
@@ -0,0 +1,159 @@
+# Functions
+
+Functions are the fundamental building block of programs. Here is the simplest
+way to make a function that adds two numbers:
+
+```python
+// Named function
+def add(x: int, y: int):
```

Comment:
In Python we try to avoid types whenever possible since that's not commonly used.

So just:
```
def add(x, y):
   return x + y
```
works